### PR TITLE
Added ConfigOptions class

### PR DESCRIPTION
**Pull request by ZakeryFyke**
Added a ConfigOptions class which takes input from the user for training and testing files, their separator types, the prediction type to be run, and the target column. Also added one validation to it to ensure the type of prediction given is valid; I'll probably add additional verification later to ensure that the given files exist, the separator is valid, and the target column is part of the given files, unless this is done by another module.

### CODE REVIEW

File: modules/DMZ/utils/config_options.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class ConfigOptions(object):
+
+    #Initialize the class
+    def __init__(self):
+        self.training_file_name = None
+        self.test_file_name = None
+        self.separator_type = None
```
Comment (ASAAR) [inline modules/DMZ/utils/config_options.py]: We don't need the separator type anymore, I auto-detect it now

Comment (asclines) [inline modules/DMZ/utils/config_options.py]: Is the auto-detect guaranteed?

File: modules/DMZ/utils/config_options.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        self.prediction_type = None
+        self.target_column = None
+        self.get_config_options()
+        self.validate_options()
+
+    #Get input from user
```
Comment (asclines) [inline modules/DMZ/utils/config_options.py]: @ZakeryFyke @ASAAR Is this how we want to get input from the user? I'm about to open a discussion issue on this with my proposed idea.

Comment (ZakeryFyke) [inline modules/DMZ/utils/config_options.py]: I'm happy to hear your ideas about it if you have something you think might be better.

Comment (asclines) [inline modules/DMZ/utils/config_options.py]: Check out #103

File: modules/DMZ/utils/config_options.py (hunk 1 above)
Comment (ASAAR) [inline modules/DMZ/utils/config_options.py]: Yes

**Pull request merged**